### PR TITLE
BZ1964063 - Adding clarification for verifying registry pods

### PR DESCRIPTION
**Pull request by kelbrown20**
For version: 4.7+
[Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1964063)

Ready for QA: @jinyunma 

Preview: 

- [Installing a cluster on vSphere with customizations](https://deploy-preview-44241--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-registry-storage-config_installing-vsphere-installer-provisioned-customizations) 
- [Installing a user-provisioned cluster on bare metal](https://deploy-preview-44241--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-registry-storage-config_installing-bare-metal)

### CODE REVIEW

**Comment by jinyunma** [comment]
@xiuwang PTAL, thanks.

File: modules/registry-configuring-storage-vsphere.adoc [hunk 1]
```diff
@@ -62,11 +62,18 @@ When using shared storage, review your security settings to prevent outside acce
 $ oc get pod -n openshift-image-registry
 ----
 +
+.Example output
+[source,yaml]
+----
+NAME                                               READY   STATUS      RESTARTS   AGE
+node-ca-7tfjt                                      1/1     Running     0          95d
+node-ca-8f82b                                      1/1     Running     0          95d
```
Comment (xiuwang) [inline modules/registry-configuring-storage-vsphere.adoc:70]: Better to use `oc get pod -n openshift-image-registry -l docker-registry=default`

```
oc get pod -n openshift-image-registry -l docker-registry=default
No resources found in openshift-image-registry namespace.
```

Comment (kelbrown20) [reply modules/registry-configuring-storage-vsphere.adoc:70]: @xiuwang Just updated, Thank you!

Comment (xiuwang) [reply modules/registry-configuring-storage-vsphere.adoc:70]: @kelbrown20 Hi, I didn't make it clear,
when use `oc get pod -n openshift-image-registry -l docker-registry=default` should return `No resources found in openshift-image-registry namespace.` when no registry pod set. But not return the node-ca pods list.

Comment (kelbrown20) [reply modules/registry-configuring-storage-vsphere.adoc]: Oh I see, my apologies for the confusion! Should the `-l docker-registry=default` still be in the documented command?

**Comment by xiuwang** [comment]
/lgtm
Thanks

**Comment by xiuwang** [comment]
@kelbrown20 Hi, I just found one issue, there is no clarification for clusteroperator status, do u need fix in this pr or need I open another bug to track?
```
Check the clusteroperator status:
$ oc get clusteroperator image-registry
```

**Comment by xiuwang** [comment]
The bug issue exists in registry-configuring-storage-baremetal.adoc , https://github.com/openshift/openshift-docs/blob/d965487967ef125df2d33828cd34ed70c8af4e63/modules/registry-configuring-storage-baremetal.adoc

**Comment by kelbrown20** [comment]
> https://github.com/openshift/openshift-docs/blob/d965487967ef125df2d33828cd34ed70c8af4e63/modules/registry-configuring-storage-baremetal.adoc

I can fix that in this PR and make a note of it in this Bugzilla. 

> The bug issue exists in registry-configuring-storage-baremetal.adoc , https://github.com/openshift/openshift-docs/blob/d965487967ef125df2d33828cd34ed70c8af4e63/modules/registry-configuring-storage-baremetal.adoc

Acknowledged! Thank you for spotting that!

**Comment by xiuwang** [comment]
We may need update the version in output per ocp version 
`oc get clusteroperator image-registry`
Example output
```
NAME                  VERSION                              AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
image-registry   4.10.6                                    True               False                    False              3h23m
```

File: modules/registry-configuring-storage-baremetal.adoc [hunk 2]
```diff
@@ -70,11 +70,16 @@ When using shared storage, review your security settings to prevent outside acce
 $ oc get pod -n openshift-image-registry
```
Comment (xiuwang) [inline modules/registry-configuring-storage-baremetal.adoc:70]: Here need update too `oc get pod -n openshift-image-registry -l docker-registry=default`

**Comment by kelbrown20** [comment]
For peer reviewers: This PR uses a version in its example output, does it have to match the product version? I didn't think so since its just an example, but best to make certain. Thank you!

File: modules/registry-configuring-storage-baremetal.adoc [hunk 3]
```diff
@@ -100,6 +105,13 @@ Leave the `claim` field blank to allow the automatic creation of an
 $ oc get clusteroperator image-registry
 ----
 +
+.Example output
+[source,yaml]
```
Comment (jeana-redhat) [inline modules/registry-configuring-storage-baremetal.adoc:109]: (I think :sweat_smile:)
```suggestion
[source,terminal]
```

File: modules/registry-configuring-storage-vsphere.adoc [hunk 4]
```diff
@@ -90,6 +95,14 @@ storage:
 ----
 $ oc get clusteroperator image-registry
 ----
++
+.Example output
+[source,yaml]
```
Comment (jeana-redhat) [inline modules/registry-configuring-storage-vsphere.adoc:100]: Same as above
```suggestion
[source,terminal]
```

**Review by jeana-redhat — COMMENTED** [review]
Think you want terminal rather than yaml for the second outputs? Otherwise LGTM

**Comment by kelbrown20** [comment]
@xiuwang Last things are updated.

**Comment by xiuwang** [comment]
/lgtm
Thanks

**Pull request merged**